docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-pytorch.rst

```diff
@@ -0,0 +1,87 @@
+Quantization-aware Training (QAT) with PyTorch
+===============================================
+
+Here, we provide the steps that are required to integrate QAT from NNCF into the training script written with
+PyTorch:
+
+
+1. Apply Post Training Quantization to the model
+####################################
+
+Quantize the model using the :doc:`Post-Training Quantization <../quantizing-models-post-training/basic-quantization-flow>` method.
+
+.. tab:: PyTorch
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_torch.py
+      :language: python
+      :fragment: [quantize]
+
+
+2. Fine-tune the model
+########################
+
+This step assumes that you will apply fine-tuning to the model the same way as it is done for the baseline model. In the
```

Comment:
There is not enough information in a brief form or a link where you can read about it, what is really happening with the model and what will be fine tuned.

Reply:
I have added some context, please check